source/renderer/app/App.js

```diff
@@ -45,10 +46,16 @@ export default class App extends Component<{
               <Fragment>
                 <Router history={history} routes={Routes} />
                 {mobxDevTools}
-                {isDaedalusDiagnosticsDialogOpen && (
+                {activeDialog === 'daedalusDiagnostics' && (
                   <DaedalusDiagnosticsDialog />
                 )}
-                {isAboutDialogOpen && <AboutDialog />}
+                {activeDialog === 'about' && <AboutDialog />}
+                {activeDialog === 'blockConsolidation' && (
+                  <BlockConsolidationStatusDialog
+                    stores={stores}
```

Comment:
@DeeJayElly are you sure we need to provide stores and actions to the container? We don't do that for other dialogs...

Reply:
@nikolaglumac done